bin/query-blocks.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Queries for the blocks on the site.
+ *
+ * @package Genesis\CustomBlocks
+ */
+
+namespace Genesis\CustomBlocks\Cli;
+
+use WP_CLI;
+use WP_Query;
+use Genesis\CustomBlocks\Blocks\Block;
+
+/**
+ * Gets the field types (controls) and their counts.
+ *
+ * @return array[] An array of associative arrays, with keys of 'field' and 'count'.
+ */
+function get_fields() {
+	$field_histogram = [];
+	$block_query     = new WP_Query(
+		[
+			'post_type'      => 'genesis_custom_block',
+			'post_status'    => 'publish',
+			'posts_per_page' => 100,
```

Comment:
The plugin [has](https://github.com/studiopress/genesis-custom-blocks/blob/54a0aba35c13c832d73aa03fae78c3c8e2ef31c3/php/Blocks/Loader.php#L141) a `posts_per_page` of `99`